CHANGELOG.md

```diff
@@ -38,6 +39,36 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
   - `Sector::from_circle`
   - `Sector::to_circle`
   - `Styled::new`
+  - `ImageRaw::new`
+  - `ImageRaw::data_width`
+  - `Image::new`
+  - `FancyDisplay::new`
+  - `MockDisplay::get_pixel`
+  - `MonoFont::mime`
+  - `MonoFont::glyph_mapping`
+  - `MonoTextStyleBuilder::new`
+  - `MonoTextStyleBuilder::text_color`
+  - `MonoTextStyleBuilder::background_color`
+  - `MonoTextStyleBuilder::underline_with_color`
+  - `MonoTextStyleBuilder::strikethrough_with_color`
+  - `MonoTextStyleBuilder::build`
+  - `Ellipse::new`
+  - `Ellipse::contains`
+  - `PrimitiveStyle::new`
+  - `PrimitiveStyle::with_stroke`
+  - `PrimitiveStyle::with_fill`
+  - `PrimitiveStyle::is_transparent`
+  - `PrimitiveStyleBuilder::new`
+  - `PrimitiveStyleBuilder::fill_color`
+  - `PrimitiveStyleBuilder::reset_fill_color`
+  - `PrimitiveStyleBuilder::stroke_color`
+  - `PrimitiveStyleBuilder::reset_stroke_color`
+  - `PrimitiveStyleBuilder::stroke_width`
+  - `PrimitiveStyleBuilder::stroke_alignment`
+  - `PrimitiveStyleBuilder::build`
+  - `DecorationColor::is_none`
+  - `DecorationColor::is_text_color`
+  - `DecorationColor::is_custom`
```

Comment:
Would it be worth condensing this list into grouped items, e.g. `Ellipse::{new, contains}` instead of listing each one separately? I don't mind either way, but a full list might be easier to <kbd>Ctrl</kbd> + <kbd>F</kbd> in.

Reply:
Suggestion:
```
  - `ImageRaw::new`, `ImageRaw::data_width`
  - `Image::new`
  - `FancyDisplay::new`
  - `MockDisplay::get_pixel`
  - `MonoFont::mime`, `MonoFont::glyph_mapping`
```
It's more compact and remains searchable.

Reply:
Changed to your suggestion. Let me know what you think. Personally the single item per line list looks cleaner, but perhaps this bikeshed is shiny enough ;)

Reply:
I think you are right, you can change it back if you like.

Reply:
Reverted 👍